db/index.js

```diff
@@ -0,0 +1,12 @@
+// const mongoose = require('mongoose');
```

Comment:
I don't think we should have commented out code. It just open doors and we can end up with a codebase where half our code is commented out and eventually it becomes clutter with unnecessary data. 

Can you add a commented out TODO instead? Same for the rest of your code.

Reply:
Agree, let's either leave a `// TODO: ` comment or remove this altogether.

Reply:
Good idea!

Reply:
can you submit your new changes, I'm still seeing your commented out code.